src/utils/optvariable.jl

```diff
@@ -77,13 +77,13 @@ given axes.
 
 # Example
 ```jldoctest
-julia> array = OptVariable{Float64}(undef, [:a, :b], 1:2);
+julia> array = OptVariable{Float}(undef, [:a, :b], 1:2);
```

Comment:
Does this work? I always thought `Int` works, but wasn't aware that `Float` also does. If so, how about replacing `Number` above also by `Float`?

Reply:
It worked for Functions, but not for the datastructs - and locally it also worked on my system, but not in the validation step. Really weird, but should be fixed now.